### PR TITLE
media player & header resizing on scroll

### PR DESCRIPTION
**Pull request by elliothwang**
media player & header resizing on scroll. Created 'scrolled' state and change inline style of player & header based on value of 'scrolled'.

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -1,15 +1,28 @@
-import React from 'react'
+import React, { useState } from 'react'
 import logo from '../images/Logo.png'
 import MediaPlayer from './MediaPlayer'
 import { Link } from 'gatsby'
 
 export default function Header() {
+  const [scrolled, setScrolled] = useState(false)
+  const changeHeader = () => {
+    window.scrollY >= 200 ? setScrolled(true) : setScrolled(false)
```
Comment (daleinen7) [inline src/components/Header.jsx:9]: Looks like this is preventing the app from deploying. We can talk about it tomorrow for a solution.

**Review by daleinen7 — CHANGES_REQUESTED** [review]

File: src/components/Header.jsx (hunk 1 above)
Comment (elliothwang) [reply src/components/Header.jsx:9]: Are you getting this error: Error: error:0308010C:digital envelope routines::unsupported? Apparently, this a fairly new error & I had to work around before changing the player. 

https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported

Comment (daleinen7) [reply src/components/Header.jsx:9]: No I haven't seen it. Maybe it's something specific to your branch?

Comment (daleinen7) [reply src/components/Header.jsx:9]: Also we never got a chance to talk over the window.scrollY. The problem is that node is reading this, and there is no `window` object in node. It seems like the consensus is if you put it inside a useEffect hook it should be fine, but I've also done something similar in my [portfolio](https://dougleinen.com). 

Here was my implmentation
```const [windowHeight, setWindowHeight] = useState()

  useEffect(() => {
    setWindowHeight(window.innerHeight)
  }, [])
```

Let me know what you find out.

**Pull request closed without merging**